test/langtools/jdk/jshell/CompletionSuggestionTest.java

```diff
@@ -324,7 +324,8 @@ public void testDocumentation() throws Exception {
                 "String(char[], int, int)");
         assertSignature("String.format(|",
                 "String String.format(String, Object...)",
-                "String String.format(java.util.Locale, String, Object...)");
+                "String String.format(java.util.Locale, String, Object...)"
```

Comment:
Nit: this change appears to be unnecessary?

Reply:
I'll revert - thanks